src/Symfony/Component/HttpFoundation/Tests/Session/Storage/Handler/MongoDbSessionHandlerTest.php

```diff
@@ -11,39 +11,46 @@
 
 namespace Symfony\Component\HttpFoundation\Tests\Session\Storage\Handler;
 
+use MongoDB\BSON\Binary;
+use MongoDB\BSON\UTCDateTime;
 use MongoDB\Client;
-use PHPUnit\Framework\MockObject\MockObject;
-use PHPUnit\Framework\SkippedTestSuiteError;
+use MongoDB\Driver\BulkWrite;
+use MongoDB\Driver\Command;
+use MongoDB\Driver\Exception\ConnectionException;
+use MongoDB\Driver\Manager;
+use MongoDB\Driver\Query;
 use PHPUnit\Framework\TestCase;
+use Symfony\Component\Clock\MockClock;
```

Comment:
The clock component needs to be added as a dev dependency.

Reply:
Good catch. The ruins of an abandoned beginning of implementation.